src/Installer/Installer.php

```diff
@@ -22,6 +22,10 @@ public static function install(string $home)
         passthru('chmod +x ' . $path . '/lib/*.sh');
         passthru('chmod +x ' . $path . '/service/*/init.sh');
 
+        passthru(sprintf('cd %s/bin/ && ln -sf ./ws-aws ./ws.aws', $path));
+        passthru(sprintf('cd %s/bin/ && ln -sf ./ws-poweroff ./ws.poweroff', $path));
+        passthru(sprintf('cd %s/bin/ && ln -sf ./ws-service ./ws.service', $path));
+
```

Comment:
what would we use the . version for after this?

Reply:
Just in case something in the harnesses use it. I don't think we do though, only `ws.aws`.